src/store/accountActions.js

```diff
@@ -3,6 +3,7 @@ import { rpc } from 'lib/rpc';
 import { getRates } from 'lib/marketApi';
 import { address } from 'lib/validators';
 import { loadTokenBalanceOf } from './tokenActions';
+import log from 'loglevel';
```

Comment:
let's comment out lines 17-22 in this file (load tokenBalance)
sorry, there was not a better place to put this comment :)